content/tutorial/02-sveltekit/02-routing/08-endpoints/README.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Endpoints
```

Comment:
Do we want to use the 'endpoints' terminology? Every other framework calls them 'API routes'. I'm not 100% sure if that's better though. I generally refer to them as '`+server.js` routes' since that's unambiguous and cements the relationship between the two concepts, but that's probably a little unwieldy. Server routes?

Reply:
If every other framework does then, I'm fine with API routes. I just know this as "endpoints" from my previous job.